presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -85,5 +82,15 @@ protected TestTable createTableWithDefaultColumns()
                         "col_required2 BIGINT NOT NULL)");
     }
 
+    @Test
+    @Override
+    public void testDelete()
```

Comment:
if `supportsDelete=true` why this test never `assertQuerySucceeds("DELETE FROM "`? 
Could you please add such scenario? The same is for other connectors.

Reply:
Added a comment below, which should answer your question

Reply:
> `// PostgreSql connector currently does not support row-by-row delete` 

Thanks for comment.
So if `supportsDelete=true` and `connector currently does not support row-by-row delete` I'd expect that this connector supports some other type of delete. I see you added `testDeleteWithSimplePredicate` and `testDeleteEntireTable`. Maybe it worth to rename this method to something like `testDeleteRowByRow`?